crud_file.c

```diff
@@ -0,0 +1,217 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#define MAX_SIZE 100
+
+// User schema structure
+struct User {
+    char id[MAX_SIZE];
+    char name[MAX_SIZE];
+    int age;
+};
+
+// Helper function to check if a user exists
+int userExists(const char *id) {
+    FILE *fp = fopen("users.txt", "r");
+    if (!fp) return 0;
+
+    char buffer[MAX_SIZE];
+    struct User user;
+
+    while (fgets(buffer, sizeof(buffer), fp)) {
+        if (sscanf(buffer, "%[^,],%[^,],%d", user.id, user.name, &user.age) == 3) {
+            if (strcmp(user.id, id) == 0) {
+                fclose(fp);
+                return 1;
+            }
+        }
+    }
+
+    fclose(fp);
+    return 0;
+}
+
+// Add a new user to the database
+void addUser() {
+    FILE *fp = fopen("users.txt", "a");
```

Comment:
Focus on variable naming which is understandable. Here, in place of "fp", a better name can be chosen.
This applies for all the variables in your code.